release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3257,17 +3257,10 @@ In the following tables, features are marked with the following statuses:
 
 * {product-title} version {product-version} disconnected installation fails on {ibm-power-name} Virtual Server. (link:https://issues.redhat.com/browse/OCPBUGS-36250[*OCPBUGS-36250*])
 
-* In {hcp} for {product-title}, if you disable the xref:../installing/overview/cluster-capabilities.adoc#ingress-operator_cluster-capabilities[Ingress capability], the Console Operator returns the following error message:
```

Comment:
This note was added in error as part of https://github.com/openshift/openshift-docs/pull/84571/files. Removing and asking Adel Zaalouk to confirm.